src/components/ha-card.ts

```diff
@@ -27,23 +27,43 @@ class HaCard extends LitElement {
         color: var(--primary-text-color);
         display: block;
         transition: all 0.3s ease-out;
+        position: relative;
       }
-      .header:not(:empty),
-      .header::slotted(*) {
+
+      :host .card-header,
```

Comment:
Is there any reason we should add `:host`? Isn't that implicit because all things in the DOM will be under `:host` ?

Reply:
You don't miss anything, do you?

Reply:
I mean that we could use `.card-header` too ?